sdk/python/lib/pulumi/output.py

```diff
@@ -536,6 +539,100 @@ def format(
             )
         return Output.from_input(format_string).apply(lambda str: str.format())
 
+    @staticmethod
+    def jsonDumps(obj: Input[object], *, skipkeys=False, ensure_ascii=True, check_circular=True,
+        allow_nan=True, cls=None, indent=None, separators=None,
+        default=None, sort_keys=False, **kw) -> "Output[str]":
```

Comment:
Do we want to use the default behavior from `json.dumps` or should we consider changing some of the defaults to give more predictable results when passing the result as an input to some other resource, e.g. having `sort_keys` default to `True`? Otherwise, if someone is passing the result of this along elsewhere, they very likely need to specify `sort_keys=True` themselves.

I know these `toJson` methods we're adding to the SDKs are "just doing what the underlying language do and aren't consistent across SDKs", so we will have different behavior across SDKs. But maybe we should still consider changing some aspects of the default behavior in each to work best with Pulumi within the language?

Reply:
idk, I flip flop each way the more I think about it. Just doing the default is easy to explain. But also having to caveat certain languages, e.g. "in Python, be sure to pass `sort_keys=True`", is also not great.

Reply:
I think I'd err to just literally being a wrapper over the language built in functionality to start and seeing if usability issues like that come up.